src/test/java/org/hydev/mcpm/utils/TemporaryDirTest.java

```diff
@@ -0,0 +1,25 @@
+package org.hydev.mcpm.utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TemporaryDirTest
+{
+    @Test
+    void test() throws IOException
+    {
+        var t = new TemporaryDir();
+        var tf = new File(t.path, "test.txt").toPath();
+        Files.writeString(tf, "meow");
+
+        assertEquals(Files.readString(tf).strip(), "meow");
+        t.close();
+        System.out.println(tf.toFile());
+        assert !tf.toFile().exists();
```

Comment:
assertFalse

Reply:
like I get why assertEquals might be useful, or assertThrows, but what can assertFalse or assertTrue tell us that assert can't?

Reply:
Consistency :eyes:

I think it gives Junit the opportunity to format the failure nicely. With an AssertionError Junit has no idea what _actually_ went wrong, just that something around this area was true instead of false.